### ISSUE TITLE
xmmintrin.h not found on arm64

### ISSUE THREAD

**Issue by bremner** [darktable-org/darktable#7583]
**Describe the bug**

```

/<<PKGBUILDDIR>>/src/common/guided_filter.h:30:10: fatal error: xmmintrin.h: No such file or directory
   30 | #include <xmmintrin.h>
      |          ^~~~~~~~~~~~~
compilation terminated.
make[3]: *** [bin/CMakeFiles/lib_darktable.dir/build.make:156: bin/CMakeFiles/lib_darktable.dir/common/box_filters.c.o] Error 1
make[3]: *** Waiting for unfinished jobs....
make[3]: Leaving directory '/<<PKGBUILDDIR>>/obj-aarch64-linux-gnu'
make[2]: *** [CMakeFiles/Makefile2:1787: bin/CMakeFiles/lib_darktable.dir/all] Error 2
```
[Full log](https://buildd.debian.org/status/fetch.php?pkg=darktablearch=arm64&ver=3.4.0-1&stamp=1608929069&raw=0)

**To Reproduce**
0. build on arm64

This is with gcc10 on Debian, but I don't think tha makes a difference.

**Expected behavior**
3.2.1 built OK on arm64, although I don't have hardware to test it on.

** Details
 - Darktable Version: 3.4.0
 - OS:  Debian GNU/Linux
 - No OpenCL
 - No GPU, AFAIK.

**Comment by parafin** [comment]
Duplicate of already fixed #7428.

**Comment by johnny-bit** [comment]
Fixed by @TurboGit in both master(3e0604a) and 3.4 branch (3e0604a2ca4739af30682d1db41ae9d241fc9b64)
